library/src/main/kotlin/wu/seal/jsontokotlin/library/Main.kt

```diff
@@ -0,0 +1,12 @@
+package wu.seal.jsontokotlin.library
+
+fun main(args: Array<String>) {
```

Comment:
What is this used for ?

Reply:
That just a temp file to check if the library integration works or not. If the integration got any issue, the program won't compile.

Reply:
Dose this file  will be compiled int o library?

Reply:
Yes it does.